docs/source/gbd2017_models/risks/smoking/index.rst

```diff
@@ -91,33 +91,25 @@ Where,
     - Note
   * - current_smoker_prevalence
     - Prevalence of current smokers
-    - /ihme/csu/swiss_re/forecast/prev_forecast_draws.csv, measure='current'
-    - Forecasted from 2020-2040
+    - :code:`J\Project\simulation_science\cancer\data\smoking\smoking_status_exposure_2019.csv`, measure='current'
+    - Province-weighted location for :ref:`Lung Cancer Screening Model <lung_cancer_cancer_concept_model>
   * - former_smoker_prevalence
     - Prevalence of former smokers
-    - /ihme/csu/swiss_re/forecast/prev_forecast_draws.csv, measure='former'
-    - Forecasted from 2020-2040
+    - :code:`J\Project\simulation_science\cancer\data\smoking\smoking_status_exposure_2019.csv`, measure='former'
+    - Province-weighted location for :ref:`Lung Cancer Screening Model <lung_cancer_cancer_concept_model>
   * - never_smoker_prevalence
     - Prevalence of never smokers
-    - 1 - current_smoker_prevalence - former_smoker_prevalence
-    - Derived from estimates forecasted from 2020-2040
+    - :code:`J\Project\simulation_science\cancer\data\smoking\smoking_status_exposure_2019.csv`, measure='never'
+    - Province-weighted location for :ref:`Lung Cancer Screening Model <lung_cancer_cancer_concept_model>
 
```

Comment:
Do these values all sum to 1, or do they need to be weighted on my end?

Reply:
Yes, all of these sum to one and are already representative of the blended province location